docker-compose.yml

```diff
@@ -18,6 +18,11 @@ services:
       - "18250:18250"
     volumes:
       - ./:/blockstore/app/:cached
+    environment:
+      - MYSQL_DATABASE=blockstore_db
+      - MYSQL_USER=root
+      - MYSQL_HOST=mysql
```

Comment:
We need to use the MySQL service (edx.devstack.mysql) that comes with the edx devstack instead of needing to install one locally.

Reply:
Yes. I am accessing the mysql service provided by the edx dev stack itself. I'm only setting the environment variables here. MySQL service is named as `mysql` in edx devstack [docker compose](https://github.com/edx/devstack/blob/master/docker-compose.yml#L64). The way to access that service is by using the name declared in the compose file as per the official [documentation](https://docs.docker.com/compose/networking/).

Reply:
Okay, I thought we would need to specify the container name. This is working.